code/dallus_content/items/clothing/donator/ties.dm

```diff
@@ -6,6 +6,34 @@
 	icon_state = "watch"
 	item_state = "watch"
 	item_color = "watch"
+	var/cooldown_timer = 300
+
+/obj/item/clothing/accessory/watch/attack_self(mob/usr)
```

Comment:
`usr` is a keyword used to reference a mob when it uses a `verb`. It can be very buggy or outright wrong when used with a proc.
Also, `attack_self()` already has a user arg anyways, so you can either put nothing in the args or replace it with `attack_self(mob/user as mob)` and then replace your `usr`s below with`user`.